src/cli_parser.cpp

```diff
@@ -537,17 +540,23 @@ void CLIParser::prepareGraph(ServerSettingsImpl& serverSettings, HFSettingsImpl&
         } else {
             serverSettings.serverMode = HF_PULL_AND_START_MODE;
         }
-
+        if (result->count("gguf_filename")) {
```

Comment:
can I use this new param with old params?
`--gguf_filename x.gguf --model_path /my/model/` is still supported? does it need to contain a graph?